.circleci/config.yml

```diff
@@ -125,6 +125,28 @@ jobs:
        - run: ci/do_circle_ci.sh check_repositories
        - run: ci/do_circle_ci.sh check_spelling
        - run: ci/do_circle_ci.sh check_spelling_pedantic
+   fuzzit_regression:
```

Comment:
@lizan should this be on AZP or CircleCI? I'm guessing AZP.

Reply:
@lizan is AZP is 100% necessary? In terms that I can see that his successfully works on CircleCI and it will require more work and changes to make it work on AZP as the current AZP `.yml` is some kind of matrix strategy which doesn't fit the current script so it will require playing with this. what are the advantages of AZP?

Reply:
@lizan I'm using the `ci/run_envoy_docker.sh 'ci/do_ci.sh bazel.fuzz'` to build the fuzzers like in the others step. where can I find the output? can I specify somehow to this script to output it into shared folder between the host and the docker?

Reply:
I'm not sure I can do everything inside the docker. does the docker on AZP has access to docker? the fuzzit CLI spawns the regression testing in docker so I guess it could work but it is a bit if inception:). but if the AZP has access to docker daemon then I guess it can work smoothly.